src/search/utils/__fixtures__/clauseQueryTestData.ts

```diff
@@ -20,6 +20,102 @@ const testData = [
       },
     ],
   },
+  {
+    description: 'should search with id if term one is present',
+    queryString: '(cc_scl_term:"SL-12345")',
+    clauses: [
+      {
+        searchTerm: {
+          id: 'id_scl_term',
+          label: 'Subcellular location term',
+          itemType: 'comment',
+          term: 'scl_term',
+          dataType: 'string',
+          hasEvidence: true,
+          autoComplete: '/uniprot/api/suggester?dict=subcell&query=?',
+          description: 'Search by comment subcellular location term',
+          example: 'membrane',
+        },
+        logicOperator: 'AND',
+        queryInput: {
+          stringValue: 'blah',
+          id: 'SL-12345',
+        },
+      },
+    ],
+  },
+  {
+    description:
+      'should append _id to term if termSuffix=true and id is present',
+    queryString: '(organism_id:"1234")',
+    clauses: [
+      {
+        searchTerm: {
+          id: 'id_organism',
+          label: 'Organism [OS]',
+          itemType: 'single',
+          term: 'organism',
+          dataType: 'string',
+          autoComplete: '/uniprot/api/suggester?dict=organism&query=?',
+          description: 'Search by Organism name',
+          example: 'saccharomyces',
+          termSuffix: true,
+        },
+        logicOperator: 'AND',
+        queryInput: {
+          stringValue: 'blah',
+          id: '1234',
+        },
+      },
+    ],
+  },
+  {
+    description:
+      'should append _name to term if termSuffix=true and id is not present',
+    queryString: '(organism_name:blah)',
+    clauses: [
+      {
+        searchTerm: {
+          id: 'id_organism',
+          label: 'Organism [OS]',
+          itemType: 'single',
+          term: 'organism',
+          dataType: 'string',
+          autoComplete: '/uniprot/api/suggester?dict=organism&query=?',
+          description: 'Search by Organism name',
+          example: 'saccharomyces',
+          termSuffix: true,
+        },
+        logicOperator: 'AND',
+        queryInput: {
+          stringValue: 'blah',
+        },
+      },
+    ],
+  },
+  {
+    description: 'should handle enzyme classification [EC] search',
```

Comment:
Wondering whether it would be worth also testing for cases where the id is not present for ec, etc...

Reply:
Good idea. Added.